test/system/220-healthcheck.bats

```diff
@@ -43,7 +43,7 @@ function _check_health {
 
     _check_health "All healthy" "
 Status           | \"healthy\"
-FailingStreak    | 0
+FailingStreak    | null
```

Comment:
ah, I think this might not be a good idea then after all.
If we hide the FailingStreak 0 even when there is a health check we could break parsers.